.github/workflows/compile_changelogs.yml

```diff
@@ -2,34 +2,51 @@ name: Compile changelogs
 
 on:
   schedule:
-    - cron: "* */12 * * *"
+  - cron: "0 0 * * *"
 
 jobs:
-  CompileCL:
-    runs-on: ubuntu-latest
-    if: github.repository == 'discordia-space/CEV-Eris'
+  compile:
+    name: "Compile changelogs"
+    runs-on: ubuntu-20.04
     steps:
-      - name: Checkout
-        uses: actions/checkout@v1
-        with: 
-          fetch-depth: 25
-      - name: Python setup
+      - name: "Check for CHANGELOG_ENABLER secret and pass true to output if it exists to be checked by later steps"
+        id: value_holder
+        env:
+          CHANGELOG_ENABLER: ${{ secrets.CHANGELOG_ENABLER }}
+        run: |
+          unset SECRET_EXISTS
+          if [ -n $CHANGELOG_ENABLER ]; then SECRET_EXISTS='true' ; fi
+          echo ::set-output name=CL_ENABLED::${SECRET_EXISTS}
+      - name: "Setup python"
+        if: steps.value_holder.outputs.CL_ENABLED
         uses: actions/setup-python@v1
         with:
           python-version: '3.x'
-      - name: Install depends
+      - name: "Install deps"
+        if: steps.value_holder.outputs.CL_ENABLED
```

Comment:
Oh, yeah, it's PR about Travis, i won't change anything in other hooks somewhy

Reply:
it's a secret var maintainer has to add on git repo, already done

Reply:
It is still off-top